data_contribution_2_data.md

```diff
@@ -0,0 +1,907 @@
+
+- Data in this directory is managed by the study team
+- It should be used as an unregulated space for intermediate and temporary files.
+- Recommend establishing retention schedules for regular review/clean-up of data in this folder.
+
+**Interim** data is different depending on the type of research. Please select what type(s) of data you would like more information about, you can select both:
+
+- [ ] omics data
+- [ ] clinical data
+<script output="data_type_interim">"@input"</script>
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_interim`)
+
+  if(data_type[0]) {
+    send.liascript(`
+    ## Omics Data
+
+![Omics data/interim](media/project_template/interim_omics.gif)
+
+This directory is for the quality control and other reporting created during a bioinformatics workflow, using the sequence files stored in the \_data/raw\_ directory.
+
+- QC metrics Quality Control (QC) metrics are reported at various stages of analysis pipelines and give information about the quality of the data generated. QC metrics files should be in a tabular file format, with .type\_metrics or .duplicate\_metrics as the extension.
+    `)
+  } else send.clear()
+} catch(e) { }
+</script>
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_interim`)
+
+  if(data_type[1]) {
+    send.liascript(`
+    ## Clinical Data
+![Clinical data/interim](media/project_template/interim_clinical.gif)
+
+This directory is for practice work generated during clincal research when analyzing and sharing original, unmodified data saved in \_data/raw\_. This can provide be a good insight into the research process and will be archived on a case by case basis. Additionaly, alternatively formatted data, or excluded data can be saved in the interim directory. Data should be saved as a tsv, csv, xml or json file if possible.
+    `)
+  } else send.clear()
+} catch(e) { }
+</script>
+
+### data/endpoints
+
+The endpoints directory holds the final results created as part of a research analysis. Often, these are files created to support papers or grants, and other dissemination. Further sub-directories can be added to organize data, if necessary.
+
+**Within an [Arcus Scientific Lab](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/Arcus_Labs_Orientation/main/arcus_orientation.md#1)**
+
+- Data in this directory is managed by the study team.
+- Data in this directory will be saved if the project is archived in Arcus.
+
+**Endpoints** data is different depending on the type of research. Please select what type(s) of data you would like more information about, you can select both:
+
+- [ ] omics data
+- [ ] clinical data
+<script output="data_type_endpoints">"@input"</script>
+
+<script modify="false">
+try {
+  let data_type = @input(`data_type_endpoints`)
+
+  if(data_type[0]) {
+    send.liascript(`
+    ## Omics Data
+
+![Omics data/endpoints](media/project_template/endpoints_omics.gif)<!-- style = "max-height: 500px;" -->
```

Comment:
Similarly, relying on an image here without copying the text into alt text or supplemental text may hinder usability in some cases. I leave it to you to decide if the effort of re-doing is worth it.